components/gemini/browser/gemini_service.cc

```diff
@@ -115,6 +115,23 @@ namespace {
     return encoded_csrf;
   }
 
+  void CalculateSaleAmount(std::string quantity,
+                           std::string price,
+                           std::string fee,
```

Comment:
Looks like these first three params could be const references.

Reply:
updated